src/components/LayoutContainer.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from "react";
+import {
+  Icon,
+  Image,
+  Menu,
+  Responsive,
+  Segment,
+  Sidebar,
+  Container
+} from "semantic-ui-react";
+
+import { Page } from "../constants";
+import logo from "../images/reel-politik-logo-1.png";
+import { string } from "prop-types";
+
+interface propTypes {
+  activePage: Page;
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+const Logo = () => (
+  <Image src={logo} style={{ height: "3em", margin: "0.5em" }} />
+);
+
+const LayoutContainer: React.SFC<propTypes> = ({
+  activePage,
+  children,
+  className,
+  style
+}): JSX.Element => {
+  const [sidebarOpen, setSidebarOpen] = useState(false);
+
+  function handleSidebarHide() {
+    setSidebarOpen(false);
+  }
+
+  function handleToggle() {
+    setSidebarOpen(!sidebarOpen);
+  }
+  return (
+    <Sidebar.Pushable {...className} {...style}>
+      <Sidebar
+        animation="overlay"
+        as={Menu}
+        inverted
+        onHide={handleSidebarHide}
+        vertical
+        visible={sidebarOpen}
+      >
+        <Menu.Item active={activePage == Page.home} className="sidebarItems">
+          Home
+        </Menu.Item>
+        <Menu.Item
```

Comment:
Is this PR supposed to include the behavior that lets me click into the "My Movies" tab?

Reply:
Nope, not yet